presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/RuleTester.java

```diff
@@ -42,6 +43,11 @@ public RuleTester()
 
     public RuleTester(Map<String, String> sessionProperties)
     {
+        this(sessionProperties, new TpchConnectorFactory(1));
+    }
+
+    public RuleTester(Map<String, String> sessionProperties, ConnectorFactory connectorFactory)
```

Comment:
Oh gosh we're going to have a merge conflict (https://github.com/Teradata/presto/pull/535/commits/7deca2c10416ed751825594913329e81e9d120d1).  Can you change this to instead take a LocalQueryRunner, and you can create a LocalQueryRunner in the test that uses your connector factory?